trin-core/src/cli.rs

```diff
@@ -129,11 +130,13 @@ pub struct TrinConfig {
     )]
     pub geth_url: Option<String>,
 
+    // todo: support custom master accs
     #[structopt(
-        long = "bridge",
-        help = "Activate to run this node in bridge mode (experimental)."
+        long = "master-accumulator-path",
+        help = "Path to master accumulator for validation (not currently supported)",
+        default_value(DEFAULT_MASTER_ACC_PATH)
     )]
-    pub bridge: bool,
+    pub master_acc_path: String,
```

Comment:
I think it is safer to convert to `PathBuf` as soon as possible, using:

 #[structopt(parse(from_os_str))]
    input: PathBuf,